CHANGES.md

```diff
@@ -1,5 +1,54 @@
 # Changes
 
+## stable-2.8.0
+
+This release introduces new a multi-cluster extension to Linkerd, allowing it
+to establish connections across Kubernetes clusters that are secure,
+transparent to the application, and work with any network topology.
+
+* The CLI has a new set of `linkerd multicluster` subcommands that provide
+  tooling to create the resources needed to discover services across
+  Kubernetes clusters.
+* The `linkerd multicluster gateways` command exposes gateway-specific
+  telemetry,  addition to the existing `stat` and `tap` commands.
+* The Linkerd-provided Grafana instance remains enabled by default, but it can
+  now be disabled; and the Linkerd dashboard can be configured to link to an
+  alternate, externally-managed Grafana instance.
+* Jaeger & OpenCensus are configurable as an [add-on][addon-2.8.0]. And the
+  proxy has been improved to emit spans with labels that reflect its pod's
+  metadata.
+* The `linkerd-cni` component has been promoted from _experimental_ to
+  _stable_.
+* `linkerd profile --open-api` now honors the `x-linkerd-retryable` and
+  `x-linkerd-timeout` OpenAPI annotations.
+* The Helm chart continues to become more flexible and modular, with new
+  Prometheus configuration options, especially. More information is available
```

Comment:
This reads a little weird. Maybe drop or move `especially`?

```suggestion
  Prometheus configuration options. More information is available
```

or

`and modular, especially with new Prometheus configuration options.`